src/data-workspace/category-combo-table/category-combo-table.js

```diff
@@ -119,6 +121,14 @@ export const CategoryComboTable = ({
         return isThisTableActive && idxDiff < headerColSpan && idxDiff >= 0
     }
 
+    const isGreyedField = (de, coc) => {
+        if (!greyedFields) {
+            return false
+        }
+        const key = getFieldId(de, coc)
+        return greyedFields[key] || false
+    }
```

Comment:
This could be extracted so the component doesn't have to create a new function every time it renders.

Reply:
You could pass `greyedFields` as a parameter to the function. It'b being called in the same component